app/src/main/java/org/mozilla/focus/search/SearchEngineAdapter.java

```diff
@@ -1,71 +0,0 @@
-/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.focus.search;
-
-import android.content.Context;
-import android.graphics.Color;
-import android.graphics.Typeface;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.BaseAdapter;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import org.mozilla.focus.R;
-
-import java.util.List;
-
-/**
- * Adapter for displaying a list of search engines.
- */
-public class SearchEngineAdapter extends BaseAdapter {
```

Comment:
We might be able to remove more:
* SearchEnginePreference
* item_search_engine.xml